Updated the README.md file for Directions and Descriptions

PythonHacker24 opened the pull request:
## Purpose

Updating the README.md file for the Project and including more descriptive information about the project and Open-Source Contribution as well as the purpose of Concordium L1LC Project.

## Changes

More descriptive approach towards explaining the project in a concise manner to provide a clear description for the aim and purpose of the project. Also updated the Installation and Setup of the project

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [*] I accept the above linked CLA.

gachouchani1999 commented on README.md:
````diff
@@ -1,55 +1,64 @@
    ```bash
-   git clone https://github.com/your-repository-url/concordium-lc1c.git
-   cd concordium-lc1c
+   git clone https://github.com/your-repository-url/concordium-local-chain.git
````
FIX URL here

PythonHacker24 replied:
Done, replaced the Link with the Official Concordium GitHub Repository

EmilLa1 commented on install.sh:
```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```
What is the purpose of this script? Surely the user would have cloned the repository already. 

I don't think we should have this script, but instead simply describe the process in the README.md file.

PythonHacker24 replied:
When the command in the README.md automated install is copied and pasted in the terminal (UNIX-based systems), it will clone the repository, build the Frontend files, Build the Rust binaries for Tauri and exit, providing the user with a file with the whole project setup. 

I am removing it as stated for manual installation.

EmilLa1 commented on README.md:
```diff
@@ -1,55 +1,73 @@
-# Concordium LC1C
+# Concordium LC1C: Local Chain 1 Click for Concordium Blockchain Development 🌐
```
I don't understand why this is changed? 

Also, we usually do not use emojis and such in READMEs, let's not start it here.

PythonHacker24 replied:
Sure will remove it as per the requirements

EmilLa1 commented on DOCUMENTATION.md:
```diff
@@ -159,6 +159,98 @@ Pool parameters are the parameters that govern the baker's pool of delegations.
 ## Troubleshooting
 There are a couple of issues that a developer might initially face on the first page when installing the tools, this section focuses on how to troubleshoot the issues and solve them. 
 
+### Rust Build dependences Issues
```
Have you tested the tool on all these platforms? 

Let's skip this section and simply spell out the prerequisites in the README. E.g. any required libraries.

PythonHacker24 replied:
Yes, all of them are tested by me on Linux and MacOS, but would fix the DOCUMENTATION.md as per the requirements and update the libraries.

EmilLa1 commented on DOCUMENTATION.md:
```diff
@@ -159,6 +159,98 @@ Pool parameters are the parameters that govern the baker's pool of delegations.
+
+Make sure that the latest version of dpkg is running on the system 
+
+### Ram Error Issues
```
Let's skip this section, and provide the link https://developer.concordium.software/en/mainnet/net/nodes/node-requirements.html#node-requirements some reasonable place in the README.

PythonHacker24 commented:
Fixed the given issues in the Documentation about the troubleshooting as well as removed the installation script and fixed the README for installation steps.
